ct.yaml

```diff
@@ -1,3 +1,8 @@
 chart-dirs:
   - charts
 validate-maintainers: false
+remote: origin
+target-branch: main
+chart-repos:
+  - stable=https://charts.helm.sh/stable
```

Comment:
are those needed? I don't see any dependencies that would require the repos

Reply:
that is true! thanks for the catch!